docs/DeveloperGuide.md

```diff
@@ -347,6 +513,9 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 ### Glossary
 
 * **Mainstream OS**: Windows, Linux, Unix, MacOS
+* **Claims**: A formal request by a client for reimbursement for losses that are covered by specific insurance plans.
+<!-- (the above definition was obtained from: https://www.iciciprulife.com/insurance-claim.html) -->
+* **Claim ID**: A formal claim ID provided by the Insurance Provider when an insurance agent submits a claim on behalf of the client.
```

Comment:
Maybe we can define what is a valid insurance plan id (insurance plan should exists in the system)